src/plugins/minimal_scene/MinimalScene.cc

```diff
@@ -1153,6 +1193,22 @@ void MinimalScene::LoadConfig(const tinyxml2::XMLElement *_pluginElem)
         renderWindow->SetCameraHFOV(fov);
       }
     }
+
+    elem = _pluginElem->FirstChildElement("view_controller");
+    if (nullptr != elem && nullptr != elem->GetText())
+    {
+      std::string viewControllerType = elem->GetText();
+      if (viewControllerType.find("orbit") != std::string::npos ||
+          viewControllerType.find("ortho") != std::string::npos)
+      {
+        renderWindow->SetCameraViewController(viewControllerType);
+      }
+      else
+      {
+        ignerr << "Unknown view controller selected: "
+               << viewControllerType << std::endl;
+      }
```

Comment:
Same as this [previous comment ](https://github.com/gazebosim/gz-gui/pull/506#discussion_r1034191859), I'd say we let `InteractiveViewControl` take care of the error handling and just set the string here

```suggestion
      renderWindow->SetCameraViewController(elem->GetText());
```